### PR TITLE
Add notes for betas for Chocolatey components

### PR DESCRIPTION
**Pull request by gep13**
## Description Of Changes

Add release notes for beta releases of all Chocolatey components

## Motivation and Context

There are releases coming up, and we need to get this documented

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [x] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue

N/A

### CODE REVIEW

File: input/en-us/agent/release-notes.md [hunk 1]
```diff
@@ -29,6 +29,33 @@ This covers the release notes for the Chocolatey Agent Service (`chocolatey-agen
...
+
+See this [list](https://github.com/chocolatey/choco/discussions/2995) for known issues with this pre-release.
+
+## Enhancement
+
+- Migrate from Rx-* packages to System.Reactive.* packages
+
+## Bug
+
+- Handle the removal of deprecated configurations from Config class in Chocolatey CLI
```
Comment (pauby) [inline input/en-us/agent/release-notes.md:56]:
```suggestion
- Handle the removal of deprecated configurations from Config class in Chocolatey CLI.
```

File: input/en-us/agent/release-notes.md [hunk 2]
```diff
@@ -29,6 +29,33 @@ This covers the release notes for the Chocolatey Agent Service (`chocolatey-agen
...
+>
+> If you run into any problems when using this beta version of Chocolatey CLI we would ask that you comment on this [discussion](https://github.com/chocolatey/choco/discussions/2995), which is where we will be collating issues, and providing workarounds, etc.  We will not be accepting issues raised against this beta release.
+
+### Known Issues
+
+See this [list](https://github.com/chocolatey/choco/discussions/2995) for known issues with this pre-release.
+
+## Enhancement
+
+- Migrate from Rx-* packages to System.Reactive.* packages
```
Comment (pauby) [inline input/en-us/agent/release-notes.md:52]:
```suggestion
- Migrate from Rx-* packages to System.Reactive.* packages.
```

File: input/en-us/choco/release-notes.md [hunk 3]
```diff
@@ -18,6 +18,49 @@ This covers changes for the "chocolatey" and "chocolatey.lib" packages, which ar
...
+- Remove WebPI alternative source - see [#2980](https://github.com/chocolatey/choco/issues/2980).
+- Rename or migrate away from custom HashCode class - see [#2909](https://github.com/chocolatey/choco/issues/2909).
+- Migrate from Rx-* packages to System.Reactive.* packages - see [#2896](https://github.com/chocolatey/choco/issues/2896).
+- Remove side-by-side installs - see [#2788](https://github.com/chocolatey/choco/issues/2788).
+- Fix spelling mistake in the extract_binary_file_from_assembly function - see [#2776](https://github.com/chocolatey/choco/issues/2776).
+- Remove deprecated Chocolatey shims (`chocolatey`, `cinst`, `clist`, `cpush`, `cuninst`, `cup`) - see [#2642](https://github.com/chocolatey/choco/issues/2642).
+- Remove deprecated configurations from Config - see [#2630](https://github.com/chocolatey/choco/issues/2630).
+- Remove the default push location when uploading/pushing packages - see [#2629](https://github.com/chocolatey/choco/issues/2629).
+- Remove Get-BinRoot Chocolatey helper function - see [#2623](https://github.com/chocolatey/choco/issues/2623).
```
Comment (pauby) [inline input/en-us/choco/release-notes.md:54]:
```suggestion
- Remove `Get-BinRoot` Chocolatey helper function - see [#2623](https://github.com/chocolatey/choco/issues/2623).
```

File: input/en-us/chocolatey-gui-licensed-extension/release-notes.md [hunk 4]
```diff
@@ -23,6 +23,30 @@ Please see [Install the Licensed Edition](xref:setup-chocolatey-gui-licensed) fo
...
+>
+> If you run into any problems when using this beta version of Chocolatey CLI we would ask that you comment on this [discussion](https://github.com/chocolatey/choco/discussions/2995), which is where we will be collating issues, and providing workarounds, etc.  We will not be accepting issues raised against this beta release.
+
+### Known Issues
+
+See this [list](https://github.com/chocolatey/choco/discussions/2995) for known issues with this pre-release.
+
+### Enhancemnet
+
+- Update to use latest beta releases of Chocolatey Components
```
Comment (pauby) [inline input/en-us/chocolatey-gui-licensed-extension/release-notes.md:47]:
```suggestion
- Update to use latest beta releases of Chocolatey Components.
```

File: input/en-us/licensed-extension/release-notes.md [hunk 5]
```diff
@@ -41,6 +41,36 @@ Please see [Install the Licensed Edition](xref:setup-licensed) for information o
...
+
+### Known Issues
+
+See this [list](https://github.com/chocolatey/choco/discussions/2995) for known issues with this pre-release.
+
+### Breaking Changes
+
+- Rename or migrate away from custom HashCode class
+- Remove side-by-side installs
+- Change default behavior of `choco list` to be local only
```
Comment (pauby) [inline input/en-us/licensed-extension/release-notes.md:66]:
```suggestion
- Rename or migrate away from custom HashCode class.
- Remove side-by-side installs.
- Change default behavior of `choco list` to be local only.
```

File: input/en-us/licensed-extension/release-notes.md [hunk 6]
```diff
@@ -41,6 +41,36 @@ Please see [Install the Licensed Edition](xref:setup-licensed) for information o
...
+
+### Bugs
+
+- Fix spelling mistake in the usage of extract_binary_file_from_assembly function
+- Handle the removal of deprecated configurations from Config class in Chocolatey CLI
```
Comment (pauby) [inline input/en-us/licensed-extension/release-notes.md:71]:
```suggestion
- Fix spelling mistake in the usage of extract_binary_file_from_assembly function.
- Handle the removal of deprecated configurations from Config class in Chocolatey CLI.
```

**Review by pauby — CHANGES_REQUESTED** [review]
Couple of typos.

**Comment by gep13** [comment]
@pauby I have fixed up your suggestions, can you have another review? Thanks!

**Review by pauby — DISMISSED** [review]
Thanks for getting those sorted @gep13. I will leave it for you to merge when you are ready.

**Pull request merged**